thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/BaselineEntity.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.thirdeye.rootcause.impl;
+
+import com.linkedin.thirdeye.rootcause.Entity;
+
+
+public class BaselineEntity extends Entity {
+  public static BaselineEntity fromURN(String urn, double score) {
```

Comment:
if it's representing timerange, have an abstract class TimeRangeEntity. CurrentEntity, BaselineTimeRangeEntity can extend the TimeRangeEntity.